backend/app/routers/aiod.py

```diff
@@ -32,6 +32,13 @@ async def get_datasets(pagination: Pagination = Depends()) -> Any:
     return await get_assets(asset_type=AssetType.DATASETS, pagination=pagination)
 
 
+@router.get("/datasets/my", response_model=list[Dataset])
```

Comment:
These my functions should also take pagination parameters, shouldnt it

Reply:
You are right.. I did not add it there as it is not supported by MyLibrary at the moment. But I can actually try to mimic it and create an GitHub issue (feature request) for it. 👍

Reply:
@marcel-vesely-kinit Pagination added, please have a look.